lib/miq_automation_engine/models/miq_ae_yaml_export.rb

```diff
@@ -259,4 +263,9 @@ def swap_domain_path(fqname)
     return fqname if @options['export_as'].blank?
     fqname.gsub(/^\/#{@domain}/, @options['export_as'])
   end
+
+  def domain_accessible?
+    return true unless @tenant
+    @tenant.ae_domains.collect(&:name).map(&:upcase).include?(@domain.upcase)
```

Comment:
Feel like this should be: `@tenant.ae_domains.any? { |d| d.name.casecmp(@domain)==0 }`